coralproject/talk
# Fix/corl 2480 correct open story drawer

=## What does this PR do?
Preserves ability to archive/unarchive stories by adding archive/unarcive button to the StoryInfoDrawer

## What changes to the GraphQL/Database Schema does this PR introduce?
None

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
- Make sure `MONGODB_ARCHIVE_URI` environment variable is set to a valid mongo uri
- log in as an admin and navigate to the Stories page
- click on the three dots icon on a story to open the StoryInfoDrawer
- if the story is currently unarchived, you should see an "Archive Story" button, click on it and see that the story is archived
- if the story is archived you should see an "unarchive story" button, click on it and see that the story is unarchived
 
 
## How do we deploy this PR?
No special considerations needed.
